### PR TITLE
Create sip-trading_incentives.md

### PR DESCRIPTION
**Pull request by kaiynne**
Draft, with header info.

When opening a pull request to submit a new SIP, please use the suggested template: https://github.com/Synthetixio/SIPs/blob/master/sip-X.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing WIP PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: SIPS/sip-63.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
...s not possible to reward traders who pay more fees on the exchange. Paying fee rebates is a very powerful way of reducing friction and switching costs for new traders. Hwoever, where CEX's can identify users and pay onboarding fees and other incentives a DEX does not have this ability so using a pooled mechanism rather than a direct rebate per user ensures the total fee incentive does not exceed a specified amount and cannot be sybil attacked. The pooled mechanism also ensures that traders who trade earlier are rewarded more than later traders incentivising traders to test out the exchange sooner. While the sX trading experience has improved significantly, there is still friction due to latency and other limitations. While we fully expect these limitations to be reduced or eliminated entirely in the near future, attracting new traders is critical to the growth of the platform. The trading experience is now sufficiently differentiated that the experience is likely to be positive relative to other DEX's particularly with respect to slippage and fees. This pooled mechanism also introduce the ability to add referal incentives and other mechanism if the base incentive proves successful.
```
Comment (justinjmoses) [inline SIPS/sip-63.md:19]: :100:

Comment (justinjmoses) [inline SIPS/sip-63.md:19]: `Hwoever` spelling

File: SIPS/sip-63.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+status: WIP
+author: Kain Warwick (@kaiynne)
+discussions-to: TBC
+
+created: 2020-05-28
+---
+
+## Simple Summary
+
+This SIP proposes a change to the Feepool contract to record the fees paid by each address to enable the distribution of trading incentives.
```
Comment (justinjmoses) [inline SIPS/sip-63.md:13]: Can you reword this to not describe how the problem will be solved (via `FeePool`) but rather that it's about incentivizing trading by distributing SNX to a pool? 

Also with the abstract - I think the details of using `FeePool` is really *how* it will be solved technically and belongs in the Technical Spec, as it's an implementation detail.

File: SIPS/sip-63.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+Feepool will be modified to write to a Trading Incentive contract each time a trade is completed. The function will check the total fees paid in that fee period and add the additional fees paid in that exchange to this balance.
+
+This fee balance will be used to calculate the distribution of the SNX during that period. At the end of the period the balance will be wiped for all addresses and fees will begin to accumulate again. Balances will be zeroed when the new SNX is deposited from the inflationary supply or manually during the trial period.
```
Comment (justinjmoses) [inline SIPS/sip-63.md:33]: Just a nitpick, but I highly doubt we will be wiping or zeroing individual address mappings - it would be too cumbersome - we'd just have a flag to indicate a change and perform a different calculation in the function. I understand what you've written is more conceptual, but we do need to be cognizant of what changes are being proposed on-chain, as some in the community may take it literally.

File: SIPS/sip-63.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+We considered a number of approaches to this problem, including minting fee tokens and paying incentives immediately on each trade. There are several benefits to this approach over the alternatives considered. The first is that the rewards are paid out on a form of bonding curve with higher rewards paid earlier when there are fewer fees accumulated in that period. The second is that users can accumulate as many fees as they want before claiming reducing gas costs and allowing even small traders to wait until they have sufficient fees to justify a withdrawal. The third is that this negates the rquirement of handling fee period rollovers or unclaimed fees. Each week the balanance of fees paid is wiped when the new SNX deposit is made ensuring all traders start each fee period with an equal chance to earn fees.
```
Comment (justinjmoses) [inline SIPS/sip-63.md:37]: spelling `rquirement`

Comment (justinjmoses) [inline SIPS/sip-63.md:37]: Good rationale :100:

File: SIPS/sip-63.md (hunk 2 above)
Comment (kaiynne) [reply SIPS/sip-63.md:13]: Fixed.

File: SIPS/sip-63.md (hunk 3 above)
Comment (kaiynne) [reply SIPS/sip-63.md:33]: Fixed, I think @jacko125 has a view on how to improve this so will keep it more generic.

File: SIPS/sip-63.md (hunk 4 above)
Comment (kaiynne) [reply SIPS/sip-63.md:37]: fixed.
